packages/ndla-ui/src/Resource/resourceComponents.tsx

```diff
@@ -73,11 +76,13 @@ const StyledTopicListElement = styled.li`
   margin: 0;
   line-height: 1.5;
   padding: 0;
-`;
-
-const StyledTopicDivider = styled.span`
-  margin: 0;
-  padding: 0 ${spacing.xxsmall};
+  display: flex;
+  align-items: center;
+  :not(:last-child):after {
+    content: '•';
+    margin: 0;
+    padding: 0 ${spacing.xxsmall};
+  }
```

Comment:
Dette var et dårlig forslag for meg, vi burde heller gå for en span med aria-hidden.